prow/images/prow-tools/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.15.5-alpine3.12 AS builder
+FROM golang:1.16.2-alpine3.13 AS builder
```

Comment:
Does the bump really is necessary? Shouldn't we bump all Go images in separate PR?

Reply:
I needed to bump this one in order to be able to remove whitelist argument form prowjobs definitions that are using program from this image